prov/tcp/src/tcpx_ep.c

```diff
@@ -65,6 +66,20 @@ void tcpx_hdr_bswap(struct tcpx_base_hdr *hdr)
 	}
 }
 
+static int tcpx_setup_socket_nodelay(SOCKET sock)
+{
+	int ret, optval = 1;
+
+	ret = setsockopt(sock, IPPROTO_TCP, TCP_NODELAY, (char *) &optval,
+			 sizeof(optval));
+	if (ret) {
+		FI_WARN(&tcpx_prov, FI_LOG_EP_CTRL,"setsockopt nodelay failed\n");
+		return ret;
+	}
+
+	return ret;
+}
```

Comment:
This change is unrelated to restricting the port value.  Either drop or move into a separate commit that explains the change.